docs/release-playbook.md

```diff
@@ -0,0 +1,203 @@
+    repository.
+
+    *   If you don't have the permission, ask one of the Buildkite org
+        admins to add you to the
+        [bazel-sheriffs](https://buildkite.com/bazel?team=bazel-sheriffs)
+        group.
+
+    *   If that worked, click "Unblock step" for the "Generate Announcement" step.
+
+*   Prepare the release announcement on http://go/bazel-newsletters.
+    *   Create a new section for the release. Populate using the generated text
+        (from the "generate announcement" step).
+    *   Reorganize the notes per category (C++, Java, etc.)
+    *   Add a comment with "+[spomorski@google.com](mailto:spomorski@google.com)" so that he takes a look.
+    *   Send an email to [bazel-dev](https://groups.google.com/forum/#!forum/bazel-dev) asking for reviewers.
+
+*   Copy & paste the generated text into a new e-mail and send it.
+    *   The first line is the recipient address.
+    *   The second line is the subject.
+    *   The rest is the body of the message.
+
+*   Trigger a new pipeline in BuildKite to test the release candidate with all the downstream projects.
+    *   Go to https://buildkite.com/bazel/bazel-with-downstream-projects-bazel
+    *   Click "New Build", then fill in the fields like this:
+        *   Message: Test Release-0.14.0rc1 (Or any message you like)
+        *   Commit: HEAD
+        *   Branch: release-0.14.0
+
+*   Look for failing projects in red.
+    *   Compare the results with the latest Bazel release:
+        *   Jobs built with the latest Bazel:
+            https://buildkite.com/bazel?team=bazel
+        *   Jobs built with release candidate: e.g.
+            https://buildkite.com/bazel/bazel-with-downstream-projects-bazel/builds/287
+
+        If a project is failing with release candidate but not with the latest
+        Bazel release, then there's probably a regression in the candidate. Ask
+        the Bazel sheriff if the problem is already noticed. If not, find out
+        the culprit, file an issue at
+        https://github.com/bazelbuild/bazel/issues and mark it as release
+        blocker.
+
+
+        If a project is failing with both release candidate and the latest Bazel
+        release, it could be a breakage from the project itself. Go through the
+        build history (eg.
+        [TensorFlow_serving](https://buildkite.com/bazel/tensorflow-serving/builds?page=2))
+        to confirm this, then file an issue to their owners.
+
+    *   File bugs (**TODO: how to find the owner/project link?**)
+
+*   Once issues are fixed, create a new candidate with the relevant cherry-picks.
+
+## Push a release
+
+1.  Verify that the following conditions **all apply**:
+    1.  at least **2 weeks passed since you pushed RC1**, and
+    1.  at least **2 business days passed since you pushed the last RC**, and
+    1.  there are **no open [release-critical Blaze bugs](https://b.corp.google.com/hotlists/14305)** in Buganizer, and
```

Comment:
Remove
Google-internal bugs are not relevant

Reply:
Done.